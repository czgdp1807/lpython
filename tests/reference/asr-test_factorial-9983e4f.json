{
    "basename": "asr-test_factorial-9983e4f",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_factorial.py",
    "infile_hash": "d379d2f378eb34c6c8e87695f9bc7c042f0047421bc231e8f0adc297",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_factorial-9983e4f.stderr",
    "stderr_hash": "f6213f13ecb220e35ac2aeb99fb2020507f4b4c8ac247b6798664ee8",
    "returncode": 2
}