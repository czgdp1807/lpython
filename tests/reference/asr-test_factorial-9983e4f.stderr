semantic error: Arguments do not match for any generic procedure, factorial
 --> tests/errors/test_factorial.py:6:11
  |
6 |     print(factorial(x))
  |           ^^^^^^^^^^^^ 
